src/styles/blocks/stars.css

```
@@ -0,0 +1,42 @@
+.stars {
+  display: flex;
+}
+
+.stars :nth-child(n) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--0 :nth-child(n) {
+  background-image: url(../../images/star.svg);
+}
+
+.stars--1 :nth-child(n + 2) {
+  background-image: url(../../images/star.svg);
+}
+
+.stars--2 :nth-child(n + 3) {
+  background-image: url(../../images/star.svg);
+}
```

```suggestion
.stars--1 :nth-child(n + 2),
.stars--2 :nth-child(n + 3) {
  background-image: url(../../images/star.svg);
}
```

You can also combine selectors with identical styles using comma